pkg/config/config.go

```diff
@@ -35,6 +35,27 @@ const (
 	NormalCosPartSizeGiB = 50
 )
 
+// refer: https://docs.harvesterhci.io/latest/settings/settings/
+func GetSystemSettingsAllowList() []string {
+	return []string{
```

Comment:
The complete list of settings is actually here: https://github.com/harvester/harvester/blob/f303b27ef7479f5727a1d6674b2d543f307575d0/pkg/settings/settings.go#L22-L45. Although some of them are not user-facing, IMO we should allow all of them to be configurable using this `SystemSettings`.

Reply:
that's also my question.
IMO, we should keep allow list only including we already fully describe in the docs.
and refuse some non user-facing settings.

Reply:
@guangbochen 
after discussing with @johnliu55tw , we decide to copy allow list from all settings including non user-facing.
plz wait for new commit before merging 
thanks a lot